esx_service/vmdk_ops.py

```diff
@@ -841,23 +853,26 @@ def main():
     try:
         port = 1019
         opts, args = getopt.getopt(sys.argv[1:], 'hp:')
-    except getopt.error, msg:
+    except getopt.error as msg:
         if msg:
-            print >> sys.stderr, msg
+           logging.exception(msg)
         usage()
-        return 1
+        return(1)
```

Comment:
actually, I do not think we should use return (val) instead of `return val`. See https://www.python.org/dev/peps/pep-0008/